schema/tables/tcc_access.yaml

```diff
@@ -0,0 +1,46 @@
+name: tcc_access
+platforms:
+  - darwin
+description: Information about macOS TCC database contents, for determining permissions granted to applications.
+columns:
+  - name: source
+    type: text
+    required: true
```

Comment:
@rachaelshaw for columns that are required, do we explicitly require the query to `select` that column, or does `select *` suffice?

Reply:
I assumed "required" meant that the column always has a value, but I could be wrong on that. @eashaw do you happen know what it means?

Reply:
@rachaelshaw It means that the column needs to be included in a `WHERE` clause for a query to return results.

Reply:
got it, then these shouldn't be required— @jacobshandling I'll update this PR

Reply:
Great, thanks @rachaelshaw @eashaw. cc @lucasmrod